### ISSUE TITLE
Weight and datasets

### ISSUE THREAD

**Issue by Mr-Kyrie** [banterle/NoR-VDPNet#4]
Hello, I recently read the paper NoR-VDPNet: A No-Reference High-Dynamic-Range Quality Metric Trained on HDR-VDP 2 and I am very interested in it. Therefore, I would like to experiment with your source code, but your GitHub code does not provide training weights and training data, so I was wondering if you could provide me with the weight files and a small part of the training data used in your paper?
Thank you very much for your help!

**Comment by banterle** [comment]
Thank you for your interest.

We are working on weights release.

>> a small part of the training data used in your paper?

The dataset is very large and it is difficult to extract a meaningful portion.
This requires a great effort.

**Comment by Mr-Kyrie** [comment]
> Thank you for your interest.
> 
> We are working on weights release.
> 
> > > a small part of the training data used in your paper?
> 
> The dataset is very large and it is difficult to extract a meaningful portion. This requires a great effort.

Thank you for your reply. Can you upload the dataset to Google Drive? Or is it possible to upload a small part of the dataset (10 images and the format of its tags) to facilitate the reproduction of your source code. Thank you very much.

**Comment by banterle** [comment]
you are welcome.

>>Can you upload the dataset to Google Drive? 
Not at the moment:
1)  We are going to release the weights.
2) The project is very large.

>> Or is it possible to upload a small part of the dataset (10 images and the format of its tags) to facilitate the 
>> reproduction of your source code

10 images are not enough, we used more than 20K images. Given that the dataset is very large, it is difficult to find 10 representative images for re-training; exploring the dataset is cumbersome. Furthermore, we had two different datasets; one for SDR images tackling distortions and one for HDR images but limited to file-size-compression. These datasets were trained separately as stated in the paper.

It is not clear your aim:
1) Do you need to run the method for comparison in a research paper?
2) Do you want to retrain with a novel dataset?

**Comment by Mr-Kyrie** [comment]
> 

1. I want to retrain with a novel datasets for  HDR images. Where can the full data be obtained? I would very much like to retrain it with your data and source code.
2. Smaller portions of the data are for debugging the code and viewing the format of the dataset.

**Comment by Mr-Kyrie** [comment]
Hi，banterle. Excuse me for interrupting you,  I would like to ask when the pre-training weights will be released?

**Comment by Mr-Kyrie** [comment]
> you are welcome.
> 
> > > Can you upload the dataset to Google Drive?
> > > Not at the moment:
> 
> 1. We are going to release the weights.
> 2. The project is very large.
> 
> > > Or is it possible to upload a small part of the dataset (10 images and the format of its tags) to facilitate the
> > > reproduction of your source code
> 
> 10 images are not enough, we used more than 20K images. Given that the dataset is very large, it is difficult to find 10 representative images for re-training; exploring the dataset is cumbersome. Furthermore, we had two different datasets; one for SDR images tackling distortions and one for HDR images but limited to file-size-compression. These datasets were trained separately as stated in the paper.
> 
> It is not clear your aim:
> 
> 1. Do you need to run the method for comparison in a research paper?
> 2. Do you want to retrain with a novel dataset?

Hi，banterle，Can you share the weights in your paper, I desperately need your weights for training, can you contact me by email if convenient?574970565@qq.com. Thank you very much

**Comment by banterle** [comment]
The weights are finishing the latest epochs, they should be ready by tomorrow. Please wait.

**Comment by banterle** [comment]
weights are online.

**Comment by Mr-Kyrie** [comment]
> weights are online.

Thanks.
